src/Microsoft.ML.LightGBM/LightGbmCatalog.cs

```diff
@@ -46,14 +46,14 @@ public static class LightGbmExtensions
         }
 
         /// <summary>
-        /// Predict a target using a decision tree regression model trained with the <see cref="LightGbmRegressorTrainer"/>.
+        /// Predict a target using a boosted decision tree regression model trained with the <see cref="LightGbmRegressorTrainer"/> and advanced options.
```

Comment:
>oosted decision tree regression [](start = 38, length = 31)

Ditto

Reply:
done

---
In reply to: [263946785](https://github.com/dotnet/machinelearning/pull/2886#discussion_r263946785) [](ancestors = 263946785)